Index: dvdisaster/dvdisaster.h
===================================================================
--- dvdisaster.orig/dvdisaster.h	2012-03-06 21:38:26.679568254 +0900
+++ dvdisaster/dvdisaster.h	2012-03-25 23:24:25.297416599 +0900
@@ -41,7 +41,6 @@
 
 #include <gtk/gtk.h>
 #include <glib/gprintf.h>
-#include <glib/gstrfuncs.h>
 
 #include <ctype.h>
 #include <errno.h>
Index: dvdisaster/memtrack.c
===================================================================
--- dvdisaster.orig/memtrack.c	2012-03-06 21:38:26.687568293 +0900
+++ dvdisaster/memtrack.c	2012-03-25 23:24:25.405417128 +0900
@@ -30,7 +30,6 @@
 
 #include <glib.h>
 #include <glib/gprintf.h>
-#include <glib/gstrfuncs.h>
 
 /*
  * We're not pulling in dvdisaster.h on purpose...
